proto/iam/rpc/session_svc.proto

```diff
@@ -35,6 +39,9 @@ service SessionSvc {
 	rpc TryStartSession (SessionSvcStartSessionRequest)	
 		returns (SessionSvcStartSessionResponse);
 		
-	rpc RefreshAccessToken (TokenRefreshRequest)	
+	rpc RefreshAccessToken (TokenRefreshRequest)
+		returns (SessionSvcStartSessionResponse);
+
+	rpc Logout(LogoutRequest)
```

Comment:
OK, hold on

Can you call this `Invalidate Refresh Token`? And give it a legitimate return type that's based on the invalidation of a refresh token?

This definitely shouldn't return `SessionSvcStartSessionResponse`...

I'd recommend writing the code and then doing the interface after the code is written, so you can work through all the error cases that might come up, e.g. "token not found", "token already invalidated", etc.